website/pages/blog/aws-encryption-and-multi-account-access.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Encryption in AWS and Multi-Account Access
+tag: security
+date: 2022/09/29
+description: >-
+  How to encrypt in AWS given a multi-account environment.
+author: jsonkao
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## *Dance like nobody is watching. Encrypt like everyone is.*
+
+ - *Werner Vogels, Amazon CTO*
+
+As AWS [outlined at their 2022 Re:Inforce Security Conference](https://www.youtube.com/watch?v=PPunA7tPMyk&t=3062s) and [mentioned by Werner Vogels at an AWS Summit in 2019](https://youtu.be/vWfkbGF6fiA?t=4339), *encrypt everything* and *encrypt like everyone is [watching]*.  
+
+![**AWS Summit Series 2019 - Santa Clara: Keynote featuring Werner Vogels**](/images/blog/aws-encryption-and-multi-account-access/encrypt-like-watching.png)
```

Comment:
Not sure if it'll work when we deploy it, but FYI these images are not rendering in the Github rendered markdown view

Reply:
Interesting, they show in the test preview.  Good eye in the markdown view!